io.vikunja.Vikunja.desktop

```
@@ -0,0 +1,9 @@
+[Desktop Entry]
+Name=Vikunja
+GenericName=to-do app
+Comment=The open-source, self-hostable to-do app. Organize everything, on all platforms.
+Type=Application
+Categories=Utility;
+Keywords=todo;task;
+Icon=io.vikunja.Vikunja
+Exec=flatpak run io.vikunja.Vikunja
```

```suggestion
Exec=vikunja
```

this always match the `command` in the manifest (more or less arguments)

---

Fixed